index.html

```diff
@@ -13424,193 +13424,346 @@ <h2>IANA Considerations</h2>
 
   <p>This section has been submitted to the Internet Engineering Steering
     Group (IESG) for review, approval, and registration with IANA.</p>
+  
+  <section>
```

Comment:
Allow predictable references to this section.

```suggestion
  <section id="media-type-ld-json">
```